aicssegmentation/core/pre_processing_utils.py

```diff
@@ -37,22 +37,26 @@ def intensity_normalization(struct_img: np.ndarray, scaling_param: List):
             struct_img[struct_img > scaling_param[0]] = struct_img.min()
         strech_min = struct_img.min()
         strech_max = struct_img.max()
-        struct_img = (struct_img - strech_min + 1e-8) / (strech_max - strech_min + 1e-8)
```

Comment:
What were these (lines 40, 47, and 55) for and why were they removed?

Reply:
Each case had its own minmax normalization step, so I just combined them all

Reply:
Oh I see that now, thanks!